docs/content/en/blog/releases/v0.25.0.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Release v0.25.0"
+linkTitle: "Release v0.25.0"
+date: 2022-02-03
+description: >
+ Release v0.25.0
+---
+
+## Changelog since v0.24.5
+
+### New Features
+
+* The event list is now available on the web console ([#3146](https://github.com/pipe-cd/pipecd/pull/3146))
+
+### Notable Changes
+
+* Make it possible to customize the variant label of Kubernetes application ([#3179](https://github.com/pipe-cd/pipecd/pull/3179))
+* Ensure that the Pipeline sync works normally even when the config file name was changed ([#3148](https://github.com/pipe-cd/pipecd/pull/3148))
+* Fix the bug of plan-preview action that caused preview comments not to be hidden ([#3157](https://github.com/pipe-cd/pipecd/pull/3157))
```

Comment:
do we need this in notable change section? looks like the below is enough 👀

Reply:
Ok. Let me update the release note later.